src/infrastructure/http/deps.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import Request
+
+from apps.shared.errors import ForbiddenError
+from infrastructure.http.domain import MindLoggerHeaders
+
+
+async def mindlogger_headers(request: Request) -> MindLoggerHeaders:
+    try:
+        mindlogger_content_source = MindLoggerHeaders(
+            mindlogger_content_source=request.headers.get(
+                "mindlogger-content-source"
+            )
+        )
+    except Exception:
```

Comment:
what kind of exception may happen, catch exact exception, not global

Reply:
Sure. I will add pydantic validations